tests/conftest.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+def pytest_addoption(parser):
```

Comment:
Could document to export `ANSIBLE_INVENTORY` which removes the need to add custom CLI arguments.

Reply:
Ah ! but I really would like pytest be aware of the inventory (check if groups are defined), so that it can bail out at the beginning. I'll find a trick (also looking at the environment variable, and prevent passing the `-i` arg in this case...)

Reply:
Exactly what I was thinking. I'm not so fond of adding arbitrary CLI arguments to existing commands.